src/resources/users/user-controllers.js

```diff
@@ -119,7 +198,7 @@ exports.delete = async (req, res) => {
 
 exports.put = async (req, res) => {
   try {
-    const updatedUser = await Users.update(req.params.id, req.body);
+    const updatedUser = await Users.update(req.body, req.body);
```

Comment:
You don't can change controllers because we are using them already on the frontend. You need to create a new controller if you dont can use this one